script/desktop/10.0/exception/index.php

```diff
@@ -1,6 +1,8 @@
 <?php
   require_once('../../../../tools/base.inc.php');
-  define('CACERT_PATH', $_SERVER['DOCUMENT_ROOT'].'/tools/cacert.pem');
+  $env = getenv();
+
+  define('CACERT_PATH', $env['DOCUMENT_ROOT'].'/tools/cacert.pem');
```

Comment:
DOCUMENT_ROOT is also a standard $_SERVER prop -- see https://www.php.net/manual/en/reserved.variables.server.php for standard props

Reply:
```suggestion
  $env = getenv();

  define('CACERT_PATH', $_SERVER['DOCUMENT_ROOT'].'/tools/cacert.pem');
```